.github/workflows/conformance-datapath.yaml

```diff
@@ -0,0 +1,268 @@
+          # addition to the image, pass the "ls" which is used as the
+          # entrypoint. The hack will go away, once @brb will create a custom
+          # GH action based on https://git.sr.ht/~motiejus/undocker.
+          # Also, we rely on the ":latest" since the datapath conformance suite
+          # is heavily WIP.
+          image: 'quay.io/cilium/cilium-cli-ci:latest ls'
+          path: '/usr/local/bin/cilium'
+      - name: Install cilium-cli
+        shell: bash
+        run: |
+          mv ${{ steps.cilium-cli.outputs.destination }}/cilium ./cilium-cli
+
+      - name: Wait for images to be available
+        timeout-minutes: 10
+        shell: bash
+        run: |
+          for image in cilium-ci operator-generic-ci hubble-relay-ci ; do
+            until docker manifest inspect quay.io/${{ github.repository_owner }}/$image:${{ steps.vars.outputs.sha }} &> /dev/null; do sleep 45s; done
+          done
+
+      - name: Run tests
+        uses: brb/lvh-action@v1.5
+        with:
+          test-name: datapath-conformance
+          image: kind
+          host-mount: ./
+          cmd: |
+            set -eux
+
+            cd /host/
+
+            ./contrib/scripts/kind.sh "" 3 "" "" "none"
+            ./cilium-cli install --wait \
+            --chart-directory=./install/kubernetes/cilium \
+            --helm-set=image.repository=quay.io/${{ github.repository_owner }}/cilium-ci \
+            --helm-set=image.useDigest=false \
+            --helm-set=image.tag=${{ steps.vars.outputs.sha }} \
+            --helm-set=operator.image.repository=quay.io/${{ github.repository_owner }}/operator \
+            --helm-set=operator.image.suffix=-ci \
+            --helm-set=operator.image.tag=${{ steps.vars.outputs.sha }} \
+            --helm-set=operator.image.useDigest=false \
+            --helm-set=hubble.relay.image.repository=quay.io/${{ github.repository_owner }}/hubble-relay-ci \
+            --helm-set=hubble.relay.image.tag=${{ steps.vars.outputs.sha }} \
+            --rollback=false \
+            --config monitor-aggregation=none \
+            --nodes-without-cilium=kind-worker3 --kube-proxy-replacement=strict --helm-set-string="k8sServiceHost=kind-control-plane,k8sServicePort=6443"
+
+            ./cilium-cli connectivity test --datapath
+
+      - name: Fetch artifacts
+        if: ${{ !success() }}
+        uses: brb/lvh-action@v1.5
+        with:
+          provision: 'false'
+          cmd: |
+            cd /host
+            kubectl get pods --all-namespaces -o wide
+            ./cilium-cli status
+            mkdir cilium-sysdumps || true
+            ./cilium-cli sysdump --output-filename cilium-sysdumps/cilium-sysdump-datapath-conformance
+
+      - name: Upload artifacts
+        if: ${{ !success() }}
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8
+        with:
+          name: cilium-sysdumps
+          path: ./cilium-sysdumps
+          retention-days: 5
```

Comment:
Not sure if this was already discussed before, but we are running this suite on `master` 4 times a day and we are keeping 5 days worth of sysdumps. If a failure occurs  in this testing suite, who is going to triage the failures? Is it `@cilium/sig-datapath`?

Reply:
TopHat is supposed to triage scheduled runs, but honestly we should just have automation tools for scheduled runs. (I plan on working on that after I'm done with the current CI rework I'm prototyping on the side.)

Reply:
> Is it @cilium/sig-datapath?

For now it's me, and once we are happy with the job, then the sig-datapath.